Fix reload endpoint and Added docs to use this endpoint (#202)
python-microservices/pyms: docs/configuration.md

```

See [Routing](routing.md) and [Examples](examples.md) to continue with this tutorial

> ## Reload configuration without stop your services

Reviewer: ```suggestion
## Reload configuration without stopping your services
```

> In production environment, you could need to change configuration files without restart the microservice/pod/container.

Reviewer: ```suggestion
In a production environment you may need to change the microservice's configuration without restarting it.
```

PyMS has a feature to reload the configuration:

```
curl -X POST http://localhost:5000/reload-config
```

> This endpoint call to the method `Microservice.reload_conf()`. This method restart the services, 

Reviewer: ```suggestion
This endpoint calls the method `Microservice.reload_conf()`, which restarts the services, 
```

> crypt configuration and initialize `create_app`.

Reviewer: ```suggestion
the encryption configuration and initializes `create_app`.
```

> cript configuration and initialize `create_app`.

Reviewer: Is `cript` a typo?

Author: yes... :see_no_evil:

```python
    def reload_conf(self):
        self.delete_services()
        self.config.reload()
        self.services = []
        self.init_services()
        self.crypt.config.reload()
        self.create_app()
```

> It means that your libraries will be restarted, for this reason, it's important to initialize your BD, 

Reviewer: ```suggestion
This means that your libraries will be restarted, which is why it's important to initialize your BD, 
```
